packages/components/src/draggable/stories/index.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Draggable from '../';
+import Dashicon from '../../dashicon';
+
+export default { title: 'Draggable', component: Draggable };
+
+const Box = ( props ) => {
```

Comment:
Yet another reason to introduce a Box component which would work cross-platform :)